Orange/widgets/visualize/owpythagoreanforest.py

```diff
@@ -44,32 +176,32 @@ class Outputs:
     depth_limit = settings.ContextSetting(10)
     target_class_index = settings.ContextSetting(0)
     size_calc_idx = settings.Setting(0)
-    zoom = settings.Setting(50)
-    selected_tree_index = settings.ContextSetting(-1)
+    zoom = settings.Setting(200)
+
+    SIZE_CALCULATION = [
+        ('Normal', lambda x: x),
+        ('Square root', lambda x: sqrt(x)),
+        ('Logarithmic', lambda x: log(x + 1)),
+    ]
+
+    @classmethod
+    def migrate_settings(cls, settings, version):
+        if version < 2:
+            del settings['selected_tree_index']
+            v1_min, v1_max = 20, 150
+            v2_min, v2_max = 100, 400
+            ratio = (v2_max - v2_min) / (v1_max - v1_min)
+            settings['zoom'] = int(settings['zoom'] * ratio + v2_min)
```

Comment:
Shouldn't this be `int(ratio * (settings['zoom'] - v1_min) + v2_min)`
Current conversion can result in values above 400.

Reply:
You're right of course, silly mistake.